cli/internal/git/utils.go

```diff
@@ -195,7 +195,7 @@ func RemoveHeadCopies(gitDirectory string) []*plumbing.Reference {
 	head, err := repo.Head()
 	if err != nil {
 		logContext.Debug(err)
-		logContext.Fatal("Failed to identify references")
+		logContext.Fatal("Failed to identify references when running `repo.Head()`")
```

Comment:
Oh I didn't see this, I won't block this but I wouldn't typically put the function called in a log like this, at least not in a fatal log. It might make more sense in a debugging log since it can help a developer figure out whats wrong but telling an end user what function was called isn't useful information to them.

@jeff-mccoy opinions?

Reply:
I'll reword it a bit, the intent was to make two different blocks have slightly different error messages. They both used to say the same thing.

Reply:
updated